frontends/web/src/routes/device/bitbox01/unlock.jsx

```diff
@@ -52,8 +54,8 @@ class Unlock extends Component {
     }
 
     focus() {
-        if (this.passwordInput) {
-            this.passwordInput.focus();
+        if (this.passwordInput.current) {
+            this.passwordInput.current.focus();
```

Comment:
I don't understand why, but this errors on the legacy BitBox01 screen

![Screen Shot 2021-12-01 at 6 26 58 PM](https://user-images.githubusercontent.com/546900/144283445-92c07995-fb7f-4431-9ad8-195cc72503b1.png)

![Screen Shot 2021-12-01 at 6 27 07 PM](https://user-images.githubusercontent.com/546900/144283466-2b41c33c-0900-4cf2-85d4-a3b92e0a6d4d.png)

Reply:
This was because i was not forwarding the ref, fixed when changed to refInput prop